main.c

```diff
@@ -321,7 +321,25 @@ static uint32_t patchTextureTable(Target target, uint32_t memory_offset, uint32_
     sprintf(path, "%s_%d_test.data", filename, i);
     printf("Loading '%s'\n", path);
     FILE* ft = fopen(path, "rb");
-    assert(ft != NULL);
+    
+    // Make sure the font data file actually opened and exit with an error if it didn't
+    if(ft == NULL){
+    	printf("One or more font data files are missing");
```

Comment:
Newline should be at end.

Reply:
You should also be specific here, which font file is missing. *Something* like `fprintf(stderr, "Unable to load \"%s\".\n", path);`